### PR TITLE
modify mark old cwd img

### PR DESCRIPTION
**Pull request by mickey8820062008**
try to modify a simple function

### CODE REVIEW

**Comment by Billy4195** [comment]
I think modifing `markExpired()` to satisfy the function in `mark_old_cwb_img` will be better, because the functions of `markOldImg()` and `markExpired()` are quite like.
And the commit subject and message should describe your purpose to change, you can refer to [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)

**Comment by icecat2012** [comment]
To modify an commit message, see the follow steps.
step1 `git log --oneline`
step2 `q`
step3 `git rebase -i XXXXXX`
step4  edit the target commit's "pick" to "reword "
step5 `ctrl+x`
step6 `Y`
step7 `Enter`
step9 Edit the commit message
step10 same as step5~7
step11 `git push -f origin BranchName`

Here is some [information](https://blog.yorkxin.org/2011/07/29/git-rebase)
ctrl+f "reword"

**Comment by icecat2012** [comment]
Hmm....
Please squash the commits.
if not sure how to do it, ask me or @Billy4195 or @chenyang14 in FB or else.

**Comment by mickey8820062008** [comment]
I am still trying to squash the commits.
I would try to ask you tomorrow.

File: dataAccessObjects.py [hunk 1]
```diff
@@ -180,7 +183,7 @@ def countUsedSchedule(self):
     def getUndecidedScheduleSn(self):
         sql = "SELECT sche_sn FROM schedule WHERE sche_id='sche0undecided' ORDER BY sche_sn ASC LIMIT 1"
         return self.queryOneValue(sql)
-
+
```
Comment (Billy4195) [inline dataAccessObjects.py]: You add one blank line with one tab, please delete it.

**Comment by mickey8820062008** [comment]
remove the tab

**Pull request closed without merging**